scripts/atlas/README.md

```diff
@@ -3,31 +3,37 @@
-All structures in the ESM Metagenomic Atlas were predicted with the ESMFold model released as `esm.pretrained.esmfold_v0()`.
+The structures in the ESM Metagenomic Atlas were predicted with `esm.pretrained.esmfold_v0()` for Atlas `v0`, while Atlas `v2023_02` used `esm.pretrained.esmfold_v1()`.
 We find that protein structures with predicted LDDT > 0.7 and predict TM > 0.7 to be both reasonably well structured and interesting.
-Therefore, we provide both the small set of "high quality" metagenomic structures, as well as the full set.
-The small set of structures is built from taking a 30% sequence identity clustering of MGnify90, and using the best structure from each cluster.
+Therefore, we provide both the small set of "high confidence" metagenomic structures from Atlas `v0`, as well as the full set.
+The small set of structures is built from taking a 30% sequence identity clustering of MGnify90 `2022_05`, and using the best structure from each cluster.
 The best structure is selected using the score pTM * pLDDT.
 
-The high quality structures are around 1TB in size.
+The high confidence structures are around 1TB in size.
 
-The full database is available as PDB structures and is 15TB in size.
+The full database is available as PDB structures and is 15TB in size for `v0`.
 
-We also provide a metadata dataframe: <https://dl.fbaipublicfiles.com/esmatlas/v0/stats.parquet>.
-You can load the file with pandas: `df = pd.read_parquet('stats.parquet')`.
-The dataframe has length `617051007`, the file size is 6.0GB and has md5 hash `3948a44562b6bd4c184167465eec17de`.
-This dataframe has 4 columns:
+We also provide a metadata dataframe: <https://dl.fbaipublicfiles.com/esmatlas/v2023_02/metadata-rc1.parquet>.
+You can load the file with pandas: `df = pd.read_parquet('metadata.parquet')`.
+The dataframe has length 773,846,840 records, and the file size is around 25GB.
+This dataframe has 9 columns:
 - `id` is the MGnify ID
 - `ptm` is the predicted TM score
 - `plddt` is the predicted average lddt
 - `num_conf` is the number of residues with plddt > 0.7
 - `len` is the total residues in the protein
+- `is_fragment` indicates whether the protein sequence is identified as a fragment in the MGnify90 sequence database.
+- `sequenceChecksum` is the CRC64 hash of the sequence. Can be used for cheaper lookups.
+- `esmfold_version` is the version of ESMFold, matching the model accessible as `esm.pretrained.esmfold_v{0,1}`
+- `atlas_version` is the Atlas version where this structure first appeared. Note: some of the predictions appearing for the first time in `v0` are also part of Atlas `v2023_02`.
 
-In parallel with `stats.parquet`, the sequences can be downloaded as fasta file from: <https://dl.fbaipublicfiles.com/esmatlas/v0/full/atlas.fasta>.
-The fasta file has `617051007` records matching the stats file, has file size 114GB, and has md5 hash `dc45f4383536c93f9d871facac7cca93`.
+Note you can remove records with missing structures with `df[~df.plddt.isna()]`.
```

Comment:
do we wanna mention proteins longer than 1281 were not folded? (optional though as I do expect few other shorter proteins did not get folded due to edge cases)

Reply:
Ah good point let me update now